src/core/ddsc/src/shm_monitor.c

```diff
@@ -80,7 +80,7 @@ dds_return_t shm_monitor_wake_and_enable(shm_monitor_t* monitor)
 dds_return_t shm_monitor_attach_reader(shm_monitor_t* monitor, struct dds_reader* reader) 
 {
 
-    if(iox_listener_attach_subscriber_event(monitor->m_listener, reader->m_iox_sub, SubscriberEvent_HAS_DATA, shm_subscriber_callback) != ListenerResult_SUCCESS) {
```

Comment:
Seems like the name of this enum value has changed.

We will need to keep track of which commit of iceoryx this iteration is compiled against.
Until a tag on both iceoryx and cyclonedds is created linking the two together, this will remain uncertain.

Reply:
It has changed and I needed it to build with the latest master. We had no release tag at this point but in the release the changed enum will be required.